scripts/pi-hole/js/network.js

```diff
@@ -136,7 +142,7 @@ $(document).ready(function() {
         "processing": true,
         "order" : [[5, "desc"]],
         "columns": [
-            {data: "ip", "type": "ip-address", "width" : "10%", "render": $.fn.dataTable.render.text() },
+            {data: "ip", "width" : "10%", "render": $.fn.dataTable.render.text(), "orderable": false },
```

Comment:
If this is no longer orderable, do we need the sorting code any more?

Reply:
The same sorting code is used on the Query Log pages.